### ISSUE TITLE
A11y: Step-by-step mode pagination 

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5107]
The last config page in step-by-step mode is displayed as “X/X”. However there is always one more page and that is displayed as “X+1/X” which looks  a little funny. 

* The pagination should account  for that page and always end at “X/X”.
* The pagination should be labelled as such. It currently seems to have no screen-readable label at all.

!image-20241014-132511.png|width=1465,height=908,alt="image-20241014-132511.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CUS-750**



[created from CUS-750, comment 503265]
